.github/policies/resourceManagement.yml

```
@@ -0,0 +1,34 @@
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
```

I recommend renaming this file to `breaking-change.yml` along with:

```suggestion
name: breaking-change
description: Automatically apply the label and comment needed to ensure breaking changes are documented properly when the 'breaking-change' label is applied.
```

Many of our repos have huge `resourceManagement.yml` files that were the result of migration from the old FabricBot service. But the new policy service bot support multiple policy configuration files, so they can be broken up into purpose-specific files.